### PR TITLE
DOCS-3301: Change paths of logos to absolute

### PR DESCRIPTION
**Pull request by Kris-MultiSafepay**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/accounting/integrations/Exact/integrations/xcore.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: "xCore"
 weight: 20
 meta_title: "Accounting integrations - xCore - MultiSafepay Docs"
-logo: '/logo/accounting/xcore.png'
+logo: 'https://raw.githubusercontent.com/MultiSafepay/docs/master/static/logo/accounting/xcore.png'
```
Comment (Sarah-MultiSafepay) [inline content/accounting/integrations/Exact/integrations/xcore.md:5]: Can you double quote this and all the other URLs listed in the logo parameter? The single quote messes with the markdown link checker.

**Review by Sarah-MultiSafepay — CHANGES_REQUESTED** [review]

File: content/accounting/integrations/Exact/integrations/xcore.md (hunk 1 above)
Comment (Kris-MultiSafepay) [reply content/accounting/integrations/Exact/integrations/xcore.md:5]: Sure thing!

**Review by Sarah-MultiSafepay — COMMENTED** [review]
There are a few places where I'm not sure we should change from single quotes to double quotes. E.g. in code excerpts. And if you really want to go all the way, you should review `meta_title` and `short_description`. :grinning:

File: content/integration/checkouts/payment-components/previous-release/previous-release-2.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ const orderData = {
 
 ```
 PaymentComponent = new MultiSafepay({
-    env: 'test',
+    env: "test",
```
Comment (Kris-MultiSafepay) [inline content/integration/checkouts/payment-components/previous-release/previous-release-2.md:57]:
```suggestion
    env: 'test',
```

**Pull request merged**